stock_location_content_template/readme/DESCRIPTION.rst

```diff
@@ -1,3 +1,5 @@
-* This module allows you to define template of location with list of products and quantities supposed to be there.
+This module allows you to define template of location with list of products and quantities supposed to be there.
```

Comment:
It does more than that.

It does not use the stock rules at all. User should be warn of that.

In a more global view, I think this will lead to extra code that will 'pollute' this repo because the behaviour is aside of Odoo.

IMHO it deserves its own repo for a 'Simple Warehouse Management'.

@OCA/logistics-maintainers What do you think ?

@max3903

Reply:
+1 on a big warning that it's quite different than one may expect. 

> IMHO it deserves its own repo for a 'Simple Warehouse Management'.

I think a different repo may be needed if there is more modules like this one.

Reply:
@hparfr https://github.com/OCA/stock-logistics-warehouse/pull/1419

Reply:
I suggest "Different wh mgnt" then.

Reply:
> I suggest "Different wh mgnt" then.

Mmmmh. 'different' in which manner?

Reply:
We can't say it's a "simple" case when the need is to manage lot of rules en masse.

Reply:
Yes, I wanted to say 'from Odoo way of doing things'.

But apart naming, to summarize, I think we agree to say this is not an extension of Odoo standard. With that assertion, we can assume a different repository is quite a sane way of doing it if we want to welcome it under OCA umbrella in order to:

- not introducing confusion for users
- gather all extension modules in the same playground
- minimize risk for testing